common/src/include/ebbrt/IOBuf.h

```diff
@@ -169,6 +169,38 @@ class IOBuf {
       return Data();
     }
 
+    // copies 'size' bytes into pointer 'vaddr'
+    void GetNoAdvance(size_t size, void* vaddr) {
+      if (!p_)
+        throw std::runtime_error("DataPointer::Get(): past end of buffer");
+
+      assert(p_->Length() > 0);
+
+      // saves vaddr pointer
+      uint8_t* temp = (uint8_t*)vaddr;
```

Comment:
no C style casts please, though this is unnecessary if you change vaddr to be a `uint8_t*`